### DISCUSSION TITLE
How to change the background color of the Room load page?

### DISCUSSION THREAD

**Discussion by Charlesc22** [mozilla/hubs#2926] [General Questions & Discussion] [answered]
I posted this question before, but it does not show up in the Discussions, so not sure what happened. So asking again here:
--------------------
How do I change the background color of the page that is shown when a Room loads? I've been able to configure everything else through the admin, but can't seem to find the setting for that one. I know it can be done as I've seen other Hubs cloud instances that have changed that background color.
------------------
for reference, this is the link to the previous time I asked it, but even though it has a number, it does not show up in discussion search
https://github.com/mozilla/hubs/discussions/2904

**Comment by emclaren** [comment]
Hi Charles, the discussions pages are in beta on Github. Weird that this one doesn't show up for some reason. I'll pass this on to the team working on it to let them know about this bug.

**Comment by misslivirose** [comment, marked as answer]
This isn't one of the standard UI elements that we surface - you will need to change this in a custom client and deploy that to a Hubs Cloud instance. It is the `var(--loading-background-color);` variable in the root-vars.scss file
